src/app/pages/class/services/class.graphql.service.ts

```diff
@@ -41,6 +41,12 @@ export class ClassService {
       variables: { name },
     });
   }
+  // classByLessonTitle(title: string) {
```

Comment:
Hey @doronbali ,
can you please remove comments that are not needed?

Reply:
done